latest_test_handler.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wptdashboard
+
+import (
+    "errors"
+    "fmt"
+    "net/http"
+    "regexp"
+    "strings"
+
+    "appengine"
+    "appengine/datastore"
+)
+
+// This handler is responsible for redirecting to the Google Cloud Storage API
+// JSON blob for the latest TestRun for the given browser.
+//
+// URL format:
+// /latest/{browser}[/{test}]
+func latestTestHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think a more accurate name would be `latestSummaryRedirectHandler`

Reply:
Done.